config/config.go

```diff
@@ -1,3 +1,16 @@
+// package config provides the ability to load and save config.json and
+// signingkeys.json files.
+//
+// Example:
+// - load config.json
+//   cfg, err := LoadConfig()
+//
+// - save config.json
+//   cfg.Save()
```

Comment:
We can remove the examples for now as those examples should be in `example_test.go` later.